src/invidious/videos/caption.cr

```diff
@@ -31,6 +31,61 @@ module Invidious::Videos
       return captions_list
     end
 
+    def timedtext_to_vtt(timedtext : String, tlang = nil) : String
```

Comment:
Here you'll need to declare your function like that in order to be able to use it outside of a class instance: `def self.timedtext_to_vtt`.

This is similar to making a function `static` in other languages (like C, C++, C#, Java, etc..)

Reply:
But I don't want to use it outside a class instance.